app/Http/Controllers/EmployeeController.php

```diff
@@ -3,29 +3,25 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
-use App\Library\MyValidation;
-use App\Http\Resources\MyCollection;
-use App\Employee;
-use App\Http\Resources\EmployeeCollection;
+use App\Services\EmployeeService;
 
 class EmployeeController extends Controller
 {
+    protected $employeeService;
+
+    function __construct()
+    {
+        $this->employeeService = new EmployeeService();
```

Comment:
- don't create new a instance on a internal class. you should pass it by constructor function.